cluster/gce/config-default.sh

```diff
@@ -201,3 +201,7 @@ ENABLE_DEFAULT_STORAGE_CLASS="${ENABLE_DEFAULT_STORAGE_CLASS:-true}"
 # TODO(dawn1107): Remove this once the flag is built into CVM image.
 # Kernel panic upon soft lockup issue
 SOFTLOCKUP_PANIC="${SOFTLOCKUP_PANIC:-false}" # true, false
+
+# Indicates if the values (eg. kube password) in metadata should be treated as
+# canonical, and therefore disk copies ought to be recreatered/clobbered.
```

Comment:
If you make any other changes, might as well s/recreatered/recreated/, although honestly I think the misspelling gives the impression of a big crater and maybe that's not all that bad :)